src/formatting_stack/strategies.clj

```diff
@@ -81,11 +80,9 @@
   The diff is compared against the `:target-branch` option."
   [& {:keys [target-branch impl files blacklist]
       :or   {target-branch "master"
-             impl          (impl/file-entries git-command "diff" "--name-status" target-branch)
+             impl          (impl/file-entries git-command "diff" "--name-only" "--diff-filter=ACMR" target-branch)
```

Comment:
If I change ACMR -> ACMRTUXB the suite also passes, which might indicate some missing coverage?

More generally, why did you pick ACMR? I'd annotate it with a comment, similarly to https://github.com/nedap/formatting-stack/blob/5c6e1f11c6ec80475063df52e1aceaded8375aee/src/formatting_stack/strategies/impl/git_diff.clj#L12

i.e. opaque bits generally call for an explanation.

Reply:
> If I change ACMR -> ACMRTUXB the suite also passes, which might indicate some missing coverage?

There's no tests for type-changed, unmerged, unknown and broken (pair) files. According to https://stackoverflow.com/a/6068691 there's no way to create files in a Unknown state.  For both broken, unmerged and type-changed files I could create a test. but it's rather cumbersome to get the repo in the exact state we need for the test. It also feels like i'm testing `git diff --diff-filter` instead of f-s.

---

I'll add a comment with the description of the PR regarding the choice